### PR TITLE
Images 3.0: Composable tools

### PR DESCRIPTION
**Pull request by parisk**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+jobs:
+  v3:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: docker/setup-buildx-action@v3
+      - uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: parisk
```
Comment (akalipetis) [inline .github/workflows/ci.yml:14]:
```suggestion
          username: sourcelair
```

**Review by akalipetis — COMMENTED** [review]
Super minor comment, apart from that this is looking 🌈 🦄 🎉 AMAZING 🎉 🦄 🌈

File: .github/workflows/ci.yml (hunk 1 above)
Comment (parisk) [reply .github/workflows/ci.yml:14]: Does not work 😞. It needs a PAT, because images are not automatically mapped with the repository.

Comment (akalipetis) [reply .github/workflows/ci.yml:14]: I thought this was used in the namespace 🤦🏽

**Review by akalipetis — APPROVED** [review]

**Pull request merged**